superset/views/schedules.py

```diff
@@ -47,6 +47,9 @@ class EmailScheduleView(
     SupersetModelView, DeleteMixin
 ):  # pylint: disable=too-many-ancestors
     include_route_methods = RouteMethod.CRUD_SET
+
+    class_permission_name = "ScheduleEmail"
```

Comment:
I am unable to see this view in the Permission set.

![Screenshot from 2020-04-03 04-28-12](https://user-images.githubusercontent.com/20752142/78307452-8f9f1100-7563-11ea-9504-e32ff6d43b03.png)